valhalla/thor/multimodal.h

```diff
@@ -57,12 +57,20 @@ class MultiModalPathAlgorithm : public PathAlgorithm {
               baldr::GraphReader& graphreader,
               const sif::mode_costing_t& mode_costing,
               const sif::TravelMode mode,
-              const Options& options = Options::default_instance());
+              const Options& options = Options::default_instance()) override;
+
+  /**
+   * Returns the name of the algorithm
+   * @return the name of the algorithm
+   */
+  virtual std::string name() const override {
```

Comment:
this should be const reference and then it should reference a static const string